scripts/cleanup-db.sh

```diff
@@ -23,7 +23,7 @@ OSRD_VALKEY="osrd-valkey"
 OSRD_VALKEY_VOLUME="osrd_valkey_data"
 OSRD_POSTGRES_PORT=5432
 OSRD_VALKEY_PORT=6379
-if [ "$PR_TEST" -eq 1 ]; then
+if [ "$PR_TEST" -eq "1" ]; then
```

Comment:
The alternative correction (for a bad type comparison when empty) is:
```sh
if [[ "$PR_TEST" -eq 1 ]]; then
```
But it casts empty `$PR_TEST` to `0` which is less maintainable IMO.

Reply:
huh? This would require bash - which the script currently doesn't, but in shell `1` and `"1"` have the same type and value, they are both the `1` *string*. Even if we then do an integer comparison for some reason (which we shouldn't anyway since it makes test spuriously error on non integer looking inputs). *The shell strongly/stringly typed nature strikes again*

If you want to fix it in this PR, I'd rather keep it POSIX and use `[ ! -z "$PR_TEST" -a ! "$PR_TEST" = 0 ]` (ie. check for unset, empty or zero)

Reply:
Indeed, I just moved to `[ "$PR_TEST"  = 1 ]` in both tests (rebased/amended)